orc8r/cloud/docker/run.py

```diff
@@ -127,6 +137,12 @@ def _parse_args() -> argparse.Namespace:
         help='Stop running containers',
     )
 
+    parser.add_argument(
+        '--clear_db', '-b',
+        action='store_true',
+        help='Clear contents of pgdata database (swagger api)',
```

Comment:
This clears the entire db, not just the data for the swagger api

Reply:
Do you know if is there any way to clear just the swagger tables?

Reply:
I would default to deleting everything -- it will have a similar effect, except that the gateways will have to re-bootstrap

But that said, (a) the first-best solution is implementing https://app.zenhub.com/workspaces/magma-5fac75d3e2cd890011f1677a/issues/magma/magma/8365, and (b) the good-enough-for-now solution is dropping specific tables. In particular, dropping all cfg_* tables should (*should* lol) have the desired effect

Reply:
Yes bootstrapping again is a desired side effect. The reason for doing that is to have the ability to start from scratch. So deleting all may be the best option here.